test/controllers/api/v2/compute_resources_controller_test.rb

```diff
@@ -498,4 +498,22 @@ def teardown
       assert_response :unprocessable_entity, "Can create libvirt compute resource with invalid name"
     end
   end
+
+  context 'ovirt' do
+    let(:simple_json) do
+      {
+        name: "oVirtTest-#{Foreman.uuid}",
+        provider: "Ovirt",
+        datacenter: "a221bee8-d53b-438e-8c43-fc9dc2b770b3",
+        url: "https://localhost/ovirt-engine/api",
+        user: "admin@internal",
+        password: "changeme"
+      }
+    end
+
+    test 'Should create oVirt resource' do
```

Comment:
This is a test without a public key, should it be moved to `ovirt_test.rb` to test creation with and without it?

Reply:
it tests to see that the resource is created using the API, but not the public key itself.
The `ovirt_test.rb` is to test the model, not the API.

Following the discussion with @bastilian, I separated the tests

Reply:
> and here you then can stub the method. Besides asserting the response you might also want to verify that public_key is actually present in the response.body

- me from our previous conversation. ;)

Reply:
@bastilian I still missing something, if I want to test my API code, and I'm creating a stub method, doesn't it miss the purpose?! 

I'm changing my code to use an existed public key under `static_fixtures`. and looking at the result now.